aws/v12.0.1/README.md

```diff
@@ -0,0 +1,23 @@
+# :zap: Giant Swarm Release v12.0.1 for AWS :zap:
+
+**If you are upgrading from 12.0.0, upgrading to this release will not roll your nodes. It will only update the apps.**
+
+## Change details
+
+#### Significant changes
+
+Cert Manager is upgraded from upstream version v0.9.0 to v0.15.2. Due to changes in Cert Manager's API,
+associated Ingresses and Secrets must also be updated to ensure they are reconciled by Cert Manager. An
+[upgrade script](https://github.com/giantswarm/cert-manager-app/blob/master/files/migrate-v090-to-v200.sh)
+is provided to assist with the process.
+
+**This is a breaking change.** Please ensure you have read the [upgrade notes](https://github.com/giantswarm/cert-manager-app#upgrading-from-v090-giant-swarm-app-v108-to-20x).
+
+**Note for SEs:** when upgrading AWS clusters after v10.0.0, ensure the App is also cordoned on the installation's
```

Comment:
What does "upgrading .. after v10" mean? Like from to where?

Reply:
the implication was that was required for nodepool clusters, but actually this makes no sense at all as this release is for nodepools anyway. I'll drop it.